### PR TITLE
German

### PR DESCRIPTION
**Pull request by ROBdk97**
More incorrect translations are fixed mainly on item names.

I will continue at a later time today

### CODE REVIEW

**Review by electronicfreak — APPROVED** [review]
better, but to much to check it all

**Comment by ROBdk97** [comment]
True xD

**Comment by ROBdk97** [comment]
Currently working on the conflicts

File: Localization/german_(germany)/global.ini [hunk 1]
```diff
@@ -52759,7 +52758,7 @@ item_NameTicket_Darkwater_Plat_01=CDF F8C Certification Pass (Platinum)
 item_NameTorshuCrab=Thorshu Grey
 item_NameTrophyCase_12Ports=Trophy Case
 item_NameUEE_medium_m_outfit=Medium Marine Armor
-item_NameUMNT_ANVL_S5_Cap=Anvil Hornet Ghost Center Cap
+item_NameUMNT_ANVL_S5_Cap=ANVIL Hornet Ghost Mittelkappe
```
Comment (Dymerz) [inline Localization/german_(germany)/global.ini:52761]: I think this should be in Title Case => "Anvil"

File: Localization/german_(germany)/global.ini [hunk 2]
```diff
@@ -56850,7 +56849,7 @@ item_SubTypeCooler_ORIG_300=Cooler ORIG 300
 item_SubTypeCooler_RSI_Aurora=Kühler RSI Aurora
 item_SubTypeCountermeasureLauncher=Abschussvorrichtung für Gegenmaßnahmen
 item_SubTypeCountermeasureLauncher_AEGS_Gladius=Gegenmaßnahme-Werfer AEGS Gladius
-item_SubTypeCountermeasureLauncher_ANVL_Hornet=Abschussgerät für Gegenmaßnahmen ANVIL Hornisse
+item_SubTypeCountermeasureLauncher_ANVL_Hornet=Abschussgerät für Gegenmaßnahmen ANVIL Hornet
```
Comment (Dymerz) [inline Localization/german_(germany)/global.ini:56852]: I think this should be in Title Case => "Anvil"

File: Localization/german_(germany)/global.ini [hunk 3]
```diff
@@ -56908,7 +56907,7 @@ item_SubTypeOxygenCap=Sauerstoff
 item_SubTypePilot=Piloten
 item_SubTypePower=Strom
 item_SubTypePower_AEGS_Gladius=Power AEGS Gladius
-item_SubTypePower_ANVL_Hornet=Power ANVIL Hornisse
+item_SubTypePower_ANVL_Hornet=Power ANVIL Hornet
```
Comment (Dymerz) [inline Localization/german_(germany)/global.ini:56910]: I think this should be in Title Case => "Anvil"

File: Localization/german_(germany)/global.ini [hunk 4]
```diff
@@ -52030,7 +52029,7 @@ item_NameMRCK_S04_ORIG_Octo_S01=SNT-481 Missile Rack
 item_NameMRCK_S04_RSI_Constellation=RSI Constellation Side Missile Rack
 item_NameMRCK_S04_VNCL_Quad_S02=Vanduul S4 Quad Missile Rack
 item_NameMRCK_S05_AEGS_Vanguard=Aegis Vanguard Torpedo Rack
-item_NameMRCK_S05_ANVL_Ballista=Anvil Ballista S05 Missile Rack
+item_NameMRCK_S05_ANVL_Ballista=ANVIL Ballista S05 Raketengestell
```
Comment (Dymerz) [inline Localization/german_(germany)/global.ini:52032]: I think this should be in Title Case => "Anvil"

File: Localization/german_(germany)/global.ini [hunk 5]
```diff
@@ -59976,7 +59975,7 @@ mission_Item_0144=Krypto-Schlüssel
 mission_client_0001=Shubin Interstellar
 mission_client_0002=ArcCorp
 mission_client_0003=Aegis-Dynamik
-mission_client_0004=ANVIL Luft- und Raumfahrt
+mission_client_0004=ANVIL Aerospace
```
Comment (Dymerz) [inline Localization/german_(germany)/global.ini:59978]: I think this should be in Title Case => "Anvil"

File: Localization/german_(germany)/global.ini [hunk 6]
```diff
@@ -66961,13 +66960,13 @@ ui_interactor_AAA_secdock_a_dockingport=Sek. Anschluss-A
 ui_interactor_AAA_secdock_b_dockingport=Sek. Anschluss-B
 ui_interactor_AAA_secdock_c_dockingport=Sek. Anschluss-C
 ui_interactor_Anniversary2019_AEGS=AEGIS Halle A
-ui_interactor_Anniversary2019_ANVL=ANVILhalle A
+ui_interactor_Anniversary2019_ANVL=ANVIL Halle A
```
Comment (Dymerz) [inline Localization/german_(germany)/global.ini:66963]: I think this should be in Title Case => "Anvil"

File: Localization/german_(germany)/global.ini [hunk 7]
```diff
@@ -68183,7 +68182,7 @@ vehicle_DescANVL_Crucible=Hersteller: Anvil Aerospace\nSchwerpunkt: Schwere Repa
...
 vehicle_DescANVL_Hawk=Hersteller: Anvil Aerospace \nFokus: Leichtes Kampfflugzeug\n\n Als kleines, leichtes Kampfflugzeug mit Schwerpunkt auf der Bewaffnung verfügt die Hawk über ein beeindruckendes Arsenal an tödlichen und nicht-tödlichen Waffen, was sie zu einem perfekten Schiff für unabhängige Kopfgeldjäger oder örtliche Sicherheitskräfte macht, die ein wenig mehr Schlagkraft suchen.
 vehicle_DescANVL_Hornet=Hersteller: Anvil Aerospace\nFokus: Mittleres Kampfflugzeug\n\nFür den Feind ist es eine nie zu unterschätzende Waffe. Für die Verbündeten ist sie ein Retter. Die F7C Hornet ist derselbe zuverlässige und widerstandsfähige Mehrzweckjäger, der zum Aushängeschild der UEE Navy geworden ist. Die F7C ist das Fundament, auf dem Sie aufbauen können, um alle Anforderungen zu erfüllen, die Sie im Sinn haben.
-vehicle_DescANVL_Hornet_DFM=ANVILhornvogel
+vehicle_DescANVL_Hornet_DFM=ANVIL Hornet
```
Comment (Dymerz) [inline Localization/german_(germany)/global.ini:68185]: I think this should be in Title Case => "Anvil"

**Comment by ROBdk97** [comment]
Fixed Anvil and Merge Conflicts

**Comment by ROBdk97** [comment]
If you wait I will commit more^^

**Comment by ROBdk97** [comment]
@Dymerz Now I translated a lot of item_Name entries and I added a few missing English entries and reordered a few entries to match english file

**Comment by ROBdk97** [comment]
I know but in this case, it was because it hadn't merged yet and I started working on it again.
Where can I create it as a draft?

**Comment by ROBdk97** [comment]
@Dymerz Oh, I found the way to convert it to a draft. 
But I'm finished now.

**Pull request merged**